tests/test_package.py

```diff
@@ -0,0 +1,4 @@
+def test_package_imports():
+    from stactools.hotosm import __version__
```

Comment:
This might be a me thing, but I don't like keeping the version in code. My preference is to specify it directly in `pyproject.toml` and then use [importlib](https://docs.python.org/3/library/importlib.metadata.html) if you really do need to find the installed version.

Reply:
I'm just happy if there's [one single source of truth](https://packaging.python.org/en/latest/discussions/single-source-version/), and in hotosm organization they seem to prefer having a `[package]/__version__.py` that is used as the source of truth for dynamic versioning within `pyproject.toml` (https://github.com/search?q=org%3Ahotosm%20__version__&type=code)

This repo was added recently as a Python package so I'm 👀 looking at that as basically a template, https://github.com/hotosm/geojson-aoi-parser

Reply:
"When in Rome" worksforme 😄